test/fixtures/dialects/postgres/postgres_create_database.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE db;
+CREATE DATABASE db OWNER user_name;
+CREATE DATABASE db OWNER = user_name;
+CREATE DATABASE db WITH OWNER = user_name;
+CREATE DATABASE db TEMPLATE template_name;
+CREATE DATABASE db TEMPLATE = template_name;
+CREATE DATABASE db WITH TEMPLATE = template_name;
+CREATE DATABASE db ENCODING 'UTF8';
+CREATE DATABASE db ENCODING = 'UTF8';
+CREATE DATABASE db LOCALE 'en_US.UTF-8';
+CREATE DATABASE db LOCALE = 'en_US.utf8';
+CREATE DATABASE db LC_COLLATE 'en_US.UTF-8' LC_CTYPE 'en_US.UTF-8';
```

Comment:
could we have a unit test with the order of `LC_COLLATE` and `LC_CTYPE` switched to cover the point we discussed in the previous review?

Reply:
I missed that!